examples/drivers/windows/otLwf/iocontrol.c

```diff
@@ -5918,20 +5918,22 @@ otLwfIoCtl_otSendActiveGet(
     *OutBufferLength = 0;
     UNREFERENCED_PARAMETER(OutBuffer);
 
-    if (InBufferLength >= sizeof(uint8_t))
+    if (InBufferLength >= sizeof(otOperationalDatasetComponents) + sizeof(uint8_t))
     {
+        const otOperationalDatasetComponents *aDatasetComp = (otOperationalDatasetComponents*)InBuffer;
         uint8_t aLength = *(uint8_t*)InBuffer;
         PUCHAR aTlvTypes = aLength == 0 ? NULL : InBuffer + sizeof(uint8_t);
```

Comment:
should here update above two lines as below?
```
uint8_t aLength = *(uint8_t*)(InBuffer + sizeof(otOperationalDatasetComponents))
PUCHAR aTlvTypes = aLength == 0 ? NULL : InBuffer + sizeof(otOperationalDatasetComponents) + sizeof(uint8_t);
```

Reply:
Yes. Thanks. Changed it.